tests/golden/defaults/cert-manager/cert-manager/50_acme_dns_vshn-default.yaml

```diff
@@ -0,0 +1,183 @@
+apiVersion: v1
+data: {}
```

Comment:
I'm surprised that ArgoCD never choked on those empty `data` fields, since we've had a lot of trouble with those for other components.